src/frontend/web_application/src/scenes/Discussion/components/MailMessage/index.jsx

```diff
@@ -81,76 +58,54 @@ class MailMessage extends Component {
     }
   }
 
-  strongSrc = '';
-  weakSrc = '';
+  formatRecipients = () => {
+    const { message, user } = this.props;
+    const prefix = (user && isUserRecipient(message, user)) ? 'Vous et ' : '';
 
-  formatRecipients = message => getRecipients(message)
-    .map(participant => participant.label).join(', ');
+    const otherRecipients = getRecipients(message)
+      .filter(participant => !isParticipantUser(participant, user))
+      .map(participant => participant.label).join(', ');
 
+    return `${prefix}${otherRecipients}`;
+  }
+
+  renderTags = ({ tags }) => (
+    tags && (
+      <ul className="s-mail-message__tags">
+        {tags.map(tag => (
+          <li key={`${this.props.message.message_id}${tag}`} className="s-mail-message__tag"><Badge>{tag}</Badge></li>
```

Comment:
> I'm not sure but I think a unique id is not mandatory this will not be on the same node tree.
It did complain about non uniqueness, this is-why I added message id